.github/BOTMETA.yml

```diff
@@ -909,106 +911,100 @@ files:
     - core inventory
     - inventory
     - inventory parsing
-  $module_utils/a10.py:
+  $module_utils/network/a10:
```

Comment:
Might be worth adding
`$module_utils/network:` as a catch all

Reply:
Indeed, for label networking and $team_networking, that would allow to remove most of the other entries TBH. We could reduce the networking part greatly thanks to the new directory structure.

Reply:
@gundalow @dagwieers 
Does that comment mean removing all other entries for label `networking` and `$team_networking` within `$module_utils/network/` and keep a single entry like
```
  $module_utils/network:
    maintainers: $team_networking
    labels: networking
```

Reply:
Correct

Reply:
Fixed